docs/source/user/api.rst

```diff
@@ -305,263 +178,90 @@ token sent in the form of a "Bearer Token", example:
+Since the detailed explanation is contained in the `Live documentation
+<#live-documentation>`_ and in the `Browsable web page <#browsable-web-interface>`_
+of each point, here we'll provide just a list of the available endpoints,
+for further information please open the URL of the endpoint in your browser.
+
+User Registration
+#################
 
 .. code-block:: text
 
   /api/v1/<organization-slug>/account/
 
-Responds only to **POST**.
-
-Parameters:
-
-===============    ===============================
-Param              Description
-===============    ===============================
-username           string
-email              string
-password1          string
-password2          string
-phone_number       string (*)
-===============    ===============================
-
-(*) ``phone_number`` is required only when the organization has enabled
-SMS verification in its "Organization RADIUS Settings".
-
 Reset password
---------------
-
-This is the classic "password forgotten recovery feature" which
-sends a reset password token to the email of the user.
+##############
 
 .. code-block:: text
 
     /api/v1/<organization-slug>/account/password/reset/
 
-Responds only to **POST**.
-
-Parameters:
-
-===============    ===============================
-Param              Description
-===============    ===============================
-email              string
-===============    ===============================
-
 Confirm reset password
-----------------------
-
-Allows users to confirm their reset password after having it requested
-via the `Reset password <#reset-password>`_ endpoint.
+######################
 
 .. code-block:: text
 
     /api/v1/<organization-slug>/account/password/reset/confirm/
 
-Responds only to **POST**.
-
-Parameters:
-
-===============    ===============================
-Param              Description
-===============    ===============================
-new_password1      string
-new_password2      string
-uid                string
-token              string
-===============    ===============================
-
 Change password
----------------
-
-**Requires the user auth token (Bearer Token)**.
-
-Allows users to change their password after using the
-`Reset password <#reset-password>`_ endpoint.
+###############
 
 .. code-block:: text
 
     /api/v1/<organization-slug>/account/password/change/
 
-Responds only to **POST**.
-
-Parameters:
-
-===============    ===============================
-Param              Description
-===============    ===============================
-new_password1      string
-new_password2      string
-===============    ===============================
-
 Login (Obtain User Auth Token)
-------------------------------
+##############################
 
 .. code-block:: text
 
     /api/v1/<organization-slug>/account/token/
 
-Responds only to **POST**.
-
-Returns:
-
-- the user radius token, which can be used to authenticate
```

Comment:
it feels wrong to me to delete good docs we have written. I would not do it. I also spent time recently on this to add docs for new endpoints. Let's leave it.